src/middlewared/middlewared/plugins/webdav.py

```diff
@@ -208,6 +209,14 @@ async def do_update(self, data):
 
         return await self.config()
 
+    @private
+    async def cert_choices(self):
+        webui_cert = await self.middleware.call('system.general.config')
+
+        filter = [["id", "!=", webui_cert['id']]] if webui_cert else []
```

Comment:
so, in case of where webdav is using the same certs as the webserver, you still pass an empty `filter` list over to `certificate.query`, right ? no ill side effects from that ?

Reply:
We are querying the config and getting it's id and not getting the certificate's id

Reply:
Yeah, whoops about the wrong id. :D